<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>626</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MarginExpand</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{background-color:#ffffff}
/*按钮的背景颜色和样式*/
#Start{
    color:black;
    border: none;
    background-color:#ffffff;
    border-radius: 5px;
    border:2px solid rgb(60, 133, 212);
	border-radius: 7px;
}




#inputFile{
    color:#ffffff;
    border: none;
    background-color:rgb(60, 133, 212);
    border-radius: 5px;
}
#outputPath{
    color:#ffffff;
    border: none;
    background-color:rgb(60, 133, 212);
    border-radius: 5px;
}

/*按钮的悬停框效果*/
#blankpaper:hover{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}
#blankpaper:checked{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}

#ruled_paper:hover{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}
#ruled_paper:checked{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}

#squredpaper:hover{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}
#squredpaper:checked{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}

#dotted_paper:hover{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}
#dotted_paper:checked{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}

#SuperNarrow:hover{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}
#SuperNarrow:checked{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}


#Narrow:hover{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}
#Narrow:checked{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}


#Wide:hover{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}
#Wide:checked{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}

#SuperWide:hover{
    border:2px solid rgb(85, 170, 255);
	border-radius: 7px;
}
#SuperWide:checked{
    border:2px solid rgb(60, 133, 212);
	border-radius: 7px;
}





</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,2,2">
     <item>
      <widget class="QTextBrowser" name="textBoard">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inputFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Output Path</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,0,5,3,0">
     <property name="spacing">
      <number>13</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1">
       <item>
        <widget class="QRadioButton" name="blankpaper">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Blank Paper</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_style</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="squredpaper">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Squared Paper</string>
         </property>
         <property name="iconSize">
          <size>
           <width>111</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_style</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ruled_paper">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Ruled Paper</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_style</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dotted_paper">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Dotted Paper</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_style</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>35</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="SuperNarrow">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Super Narrow(20%)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_size</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Narrow">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Narrow(30%)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_size</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Wide">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wide(50%)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_size</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="SuperWide">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Super Wide(100%)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_size</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Start">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start！</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>3</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_size">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="buttonGroup_style"/>
 </buttongroups>
</ui>
